hydra-tui/src/Hydra/TUI/Model.hs

```diff
@@ -200,12 +200,13 @@ newActiveLink parties headId =
 
 isModalOpen :: RootState -> Bool
 isModalOpen s =
-  case s
-    ^? connectedStateL
-      . connectionL
-      . headStateL
-      . activeLinkL
-      . activeHeadStateL
-      . openStateL of
-    Just OpenHome -> False
-    _ -> True
+  case connectedState s of
+    Disconnected -> False
+    cs -> case cs
+      ^? connectionL
+        . headStateL
+        . activeLinkL
+        . activeHeadStateL
+        . openStateL of
+      Just OpenHome -> False
+      _ -> True
```

Comment:
I think this does not fix it completely. It's not only the `connectedState` that could yield `Nothing`, but also the `activeLinkL` and `openStateL` (if it is not in open state).

As the `OpenState` is the only one that has a modal (all other constructors than `OpenHome`, I think we should just pattern match on `Nothing` and handle it that way:

```suggestion
  case s
    ^? connectedStateL
      . connectionL
      . headStateL
      . activeLinkL
      . activeHeadStateL
      . openStateL of
    Nothing -> False
    Just OpenHome -> False
    Just _ -> True
```